packages/expo-permissions/ios/EXPermissions/EXPermissions.m

```diff
@@ -208,10 +208,7 @@ + (UMPermissionStatus)statusForPermission:(NSDictionary *)permission
 
 - (id<UMPermissionsRequester>)getPermissionRequesterForType:(NSString *)type
 {
-  static dispatch_once_t once;
-  dispatch_once(&once, ^{
-    [self ensureRequestersFallbacksAreRegistered];
-  });
+  [self ensureRequestersFallbacksAreRegistered];
```

Comment:
Good catch! If we change the code like you propose we will call `ensureRequestersFallbacksAreRegistered` every time a requester is… requested, which may slightly impact performance and is unneeded.

Why don't we just move the `dispatch_once_t` to an instance variable, so that the `ensure` is called once for every instance of `EXPermissions` (that's what I wanted to achieve from the beginning 😞)?